icu4c/source/data/misc/units.txt

```diff
@@ -374,4 +374,1408 @@ units:table(nofallback){
+                        unit{"meter-and-centimeter"}
+                    }
+                }
+                NO{
+                    {
+                        unit{"meter-and-centimeter"}
+                    }
+                }
+                PL{
+                    {
+                        unit{"meter-and-centimeter"}
+                    }
+                }
+                PT{
+                    {
+                        unit{"meter-and-centimeter"}
+                    }
+                }
+                RU{
+                    {
+                        unit{"meter-and-centimeter"}
+                    }
+                }
+                SA{
+                    {
+                        unit{"meter-and-centimeter"}
+                    }
+                }
+                SE{
+                    {
+                        unit{"meter-and-centimeter"}
+                    }
+                }
+                TR{
+                    {
+                        unit{"meter-and-centimeter"}
+                    }
+                }
+                US{
+                    {
+                        geq{"3.0"}
+                        unit{"foot-and-inch"}
+                    }
+                }
+                VN{
+                    {
+                        unit{"meter-and-centimeter"}
+                    }
+                }
+            }
+            "rainfall"{
+                001{
+                    {
+                        unit{"millimeter"}
+                    }
+                }
+                BR{
+                    {
+                        unit{"centimeter"}
+                    }
+                }
+                US{
+                    {
+                        unit{"inch"}
+                    }
+                }
+            }
+            "road"{
+                001{
+                    {
+                        geq{"0.9"}
+                        unit{"kilometer"}
+                    }
+                    {
+                        geq{"300.0"}
+                        skeleton{"precision-increment/50"}
+                        unit{"meter"}
+                    }
+                    {
+                        skeleton{"precision-increment/10"}
+                        unit{"meter"}
+                    }
+                    {
+                        unit{"meter"}
+                    }
+                }
+                GB{
+                    {
+                        geq{"0.5"}
+                        unit{"mile"}
+                    }
+                    {
+                        geq{"100.0"}
+                        skeleton{"precision-increment/50"}
+                        unit{"yard"}
+                    }
+                    {
+                        unit{"yard"}
+                    }
+                }
+                SE{
+                    {
+                        geq{"0.1"}
+                        unit{"mile-scandinavian"}
```

Comment:
We may need to add smaller units to SE.

Reply:
This would be a CLDR change though, so considered out of scope for this pull request.